libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -570,9 +719,16 @@ def __SshConnection(self) -> None:
 
     devnull = open(os.devnull, 'w')
     subprocess.check_call([
-        'gcloud', 'compute', '--project', self.project_id, 'ssh',
-        '--zone', self.zone, self.name
-    ], stderr=devnull)
+        'gcloud',
+        'compute',
+        '--project',
+        self.project_id,
+        'ssh',
+        '--zone',
+        self.zone,
+        self.name
+    ],
+                          stderr=devnull)
```

Comment:
plz fix indentation here. Might be a good idea to build the command list in one variable and have the full `check_call` call in one line

Reply:
Done